src/Raven.Pal/src/posix/linuxonly.c

```diff
@@ -108,5 +109,39 @@ rvn_test_storage_durability(
     return rc;
 }
 
+PRIVATE int32_t 
+_read_sysfs_file_stat(char *filename, struct IO_STATS *ios)
+{
+	FILE *fp;
+	int i;
+	unsigned long rd_ios, wr_ios, rd_sec_or_wr_ios;
+
+	/* Try to read given stat file */
+	if ((fp = fopen(filename, "r")) == NULL)
+		return -1;
+
+	i = fscanf(fp, "%lu %*u %lu %*u %lu",
```

Comment:
what is the structure you expect here?

Reply:
https://github.com/ravendb/ravendb/pull/14129#discussion_r859704752